tools/chef/roles/db-mysql56-server.json

```diff
@@ -17,12 +33,14 @@
         "packages": [
           "Percona-Server-devel-56",
           "Percona-Server-server-56"
-        ]
+        ],
+        "service_name": "mysql"
```

Comment:
the seed starts with CentOS 7, so the default config will need it to to be mysqld